packages/sanity/src/core/releases/store/createReleaseStore.ts

```diff
@@ -46,7 +46,7 @@ const QUERY = `*[${QUERY_FILTER}] ${QUERY_PROJECTION} | ${QUERY_SORT_ORDER}`
 
 const INITIAL_STATE: ReleasesReducerState = {
   releases: new Map(),
-  state: 'loaded' as const,
+  state: 'initialising' as const,
```

Comment:
Just want to double check: this is intentional right?

Reply:
Yes, this is intentional as loading was wrongly coming back as false initially and then immediately to true which caused a few minor loading flickers on the releases list